spec/jobs/facilities/dental_service_job_reload_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'rails_helper'
+
+RSpec.describe Facilities::DentalServiceReloadJob, type: :job do
+  let(:dental_service_data) do
+    %(unique_id,facility_type\n402HB,va_health_facility\n436,va_health_facility\n436GH,va_health_facility)
+  end
+
+  before(:each) do
+    allow_any_instance_of(
+      Facilities::DentalServiceReloadJob
+    ).to receive(:fetch_dental_service_data).and_return(CSV.parse(dental_service_data, headers: true))
+  end
+
+  it 'populates facilities' do
+    Facilities::DentalServiceReloadJob.new.perform
+    expect(FacilityDentalService.find('402HB')).to_not be_nil
+    expect(FacilityDentalService.find('436')).to_not be_nil
+    expect(FacilityDentalService.find('436GH')).to_not be_nil
+  end
+
+  it 'populates facility data' do
+    now = Time.now.utc.iso8601
+    Facilities::DentalServiceReloadJob.new.perform
+    expect(FacilityDentalService.find('402HB').local_updated).to be >= now
+  end
+
+  it 'deletes removed keys' do
+    Facilities::DentalServiceReloadJob.new.perform
+    expect(FacilityDentalService.find('436GH')).to_not be_nil
+
+    dental_service_data = %(unique_id,facility_type\n402HB,va_health_facility\n436,va_health_facility)
+
+    allow_any_instance_of(
+      Facilities::DentalServiceReloadJob
+    ).to receive(:fetch_dental_service_data).and_return(CSV.parse(dental_service_data, headers: true))
+
+    Facilities::DentalServiceReloadJob.new.perform
+    expect(FacilityDentalService.find('436GH')).to be_nil
+  end
+
+  context 'on error' do
+    before do
```

Comment:
I think you can do something like this:

```rb
require 'lib/sentry_logging_spec_helper'

before do
  let(:exception) { Facilities::DentalServiceError }
end

it_behaves_like 'a sentry logger'
```

Though the shared example might not be robust enough to just drop in here, yet

Reply:
I changed the spec to have this in another branch, but learned that those group specs will test the exception itself, but not the code calling the exception. Changing this test reduced the code coverage and made simplecov unhappy. So I am leaving it how it is for now